### PR TITLE
Added 3p oath TOTP apps as options

### PR DESCRIPTION
**Pull request by Sammak**

### CODE REVIEW

File: microsoft-365/business-premium/m365bp-conditional-access.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ This article provides information about:
 
 Security defaults were designed to help protect your company's user accounts from the start. When turned on, security defaults provide secure default settings that help keep your company safe by:
 
-- Requiring all users and admins to register for MFA using the Microsoft Authenticator app.
+- Requiring all users and admins to register for MFA using the (Microsoft Authenticator app)[../authentication/concept-authentication-authenticator-app.md] or any third-party application using (OATH TOTP)[../authentication/concept-authentication-oath-tokens.md#oath-software-tokens].
```
Comment (JohanFreelancer9) [inline microsoft-365/business-premium/m365bp-conditional-access.md]:
```suggestion
- Requiring all users and admins to register for MFA using the [Microsoft Authenticator app](../authentication/concept-authentication-authenticator-app.md) or any third-party application using [OATH TOTP](../authentication/concept-authentication-oath-tokens.md#oath-software-tokens).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Sammak, kindly commit the suggested changes. Many thanks.

**Comment by Sammak** [comment]
@JohanFreelancer9 Done, thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by denisebmsft — APPROVED** [review]
Thank you @Sammak and @JohanFreelancer9

**Pull request merged**